api/src/features/todo/use_cases/get_todo_by_id.py

```diff
@@ -1,26 +1,14 @@
 from typing import cast
 
-from pydantic import BaseModel, Field
-
 from common.exceptions import MissingPrivilegeException
 from data_providers.repository_interfaces.TodoRepositoryInterface import (
     TodoRepositoryInterface,
 )
-from entities.TodoItem import TodoItem
-
-
-class GetTodoByIdResponse(BaseModel):
-    id: str = Field(...)
-    title: str = Field(...)
-    is_completed: bool = False
-
-    @staticmethod
-    def from_entity(todo_item: TodoItem) -> "GetTodoByIdResponse":
-        return GetTodoByIdResponse(id=todo_item.id, title=todo_item.title, is_completed=todo_item.is_completed)
+from features.todo.shared_models import TodoItemResponseModel
 
 
-def get_todo_by_id_use_case(id: str, user_id: str, todo_repository: TodoRepositoryInterface) -> GetTodoByIdResponse:
+def get_todo_by_id_use_case(id: str, user_id: str, todo_repository: TodoRepositoryInterface) -> TodoItemResponseModel:
     todo_item = todo_repository.get(id)
     if todo_item.user_id != user_id:
         raise MissingPrivilegeException
-    return GetTodoByIdResponse.from_entity(cast(TodoItem, todo_item))
+    return TodoItemResponseModel.parse_obj(cast(TodoItemResponseModel, todo_item))
```

Comment:
What does the `cast` function _actually_ do here?

Reply:
should not be there, fixed